internal/resources/schemas/lb_profile_schemas.go

```diff
@@ -0,0 +1,450 @@
+				"persistence_entry_timeout": {
+					Type:     schema.TypeInt,
+					Optional: true,
+					//Default:          300,
+					//ValidateDiagFunc: validations.IntBetween(1, 2147483647),
+					Description: "persistence_entry_timeout for Network Load balancer Profile",
+				},
+			},
+		},
+	}
+}
+
+func ClientProfileSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:        schema.TypeList,
+		Optional:    true,
+		Description: "Client Profile configuration",
+		MaxItems:    1,
+		ExactlyOneOf: []string{"http_profile", "tcp_profile", "udp_profile",
+			"cookie_profile", "sourceip_profile", "generic_profile",
+			"client_profile", "server_profile"},
+		ConflictsWith: []string{"http_profile", "tcp_profile",
+			"udp_profile", "cookie_profile", "sourceip_profile",
+			"generic_profile", "server_profile"},
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"profile_type": {
+					Type: schema.TypeString,
+					ValidateDiagFunc: validations.StringInSlice([]string{
+						"application-profile", "ssl-profile", "persistence-profile",
+					}, false),
+					Optional:    true,
+					Default:     "ssl-profile",
+					Description: "Network Loadbalancer Supported values are `application-profile`, `ssl-profile`, `persistence-profile`",
+				},
+				"ssl_suite": {
+					Type:             schema.TypeString,
+					ValidateDiagFunc: validations.StringInSlice([]string{"BALANCED", "HIGH_SECURITY", "HIGH_COMPATIBILITY", "CUSTOM"}, false),
+					Optional:         true,
+					Default:          "BALANCED",
+					Description:      "Network Loadbalancer Supported values are `BALANCED`,`HIGH_SECURITY`, `HIGH_COMPATIBILITY`,`CUSTOM`",
+				},
+				"session_cache": {
+					Type:        schema.TypeBool,
+					Optional:    true,
+					Default:     true,
+					Description: "session_cache for Network Load balancer Profile",
+				},
+				"session_cache_entry_timeout": {
+					Type:        schema.TypeInt,
+					Optional:    true,
+					Default:     300,
+					Description: "session_cache_entry_timeout for Network Load balancer Profile",
+				},
+				"prefer_server_cipher": {
+					Type:        schema.TypeBool,
+					Optional:    true,
+					Default:     true,
+					Description: "prefer_server_cipher for Network Load balancer Profile",
+				},
+			},
+		},
+	}
+}
+
+func ServerProfileSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:        schema.TypeList,
+		Optional:    true,
+		Description: "Server Profile configuration",
+		MaxItems:    1,
+		ExactlyOneOf: []string{"http_profile", "tcp_profile", "udp_profile",
+			"cookie_profile", "sourceip_profile", "generic_profile",
+			"client_profile", "server_profile"},
+		ConflictsWith: []string{"http_profile", "tcp_profile",
+			"udp_profile", "cookie_profile", "sourceip_profile",
+			"generic_profile", "client_profile"},
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"profile_type": {
+					Type: schema.TypeString,
+					ValidateDiagFunc: validations.StringInSlice([]string{
```

Comment:
I hope you are taking care of fields that cannot be updated post creation.

Reply:
updated you can get the validation error in above screenshot